Library/ViewModels/PledgeViewModel.swift

```diff
@@ -352,7 +352,7 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .switchMap { input in
         AppEnvironment.current.apiService.updateBacking(input: input)
           .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
-          .map { $0 as StripeSCARequiring }
+          .map { ($0.updateBacking.checkout.id, $0 as StripeSCARequiring) }
```

Comment:
Why do we need to map out the checkout id in the `updateBacking` case? 🤔

Reply:
Because `createOrUpdateEvent` requires that `createBackingEvents` and `updateBackingEvents` have the same type (<(String, StripeSCARequiring), GraphError>.Event>) . So I saw some options here: 
1- map out checkout id even if we won't use it
2- make the string of the tuple optional in both `createBackingEvents` and `updateBackingEvents` signals

Tried other solutions but wasn't able to make the compiler happy.

Reply:
Hmm I think just mapping the value in `createOrUpdateEvent` seems to work and doesn't require mapping out the `checkoutId` in the update events:

```
    let createOrUpdateEvent = Signal.merge(
      createBackingEvents.map { $0.map(second) },
      updateBackingEvents
    )
```

Reply:
I don't know why I didn't think of that 🤦🏻‍♂️ Thanks, @ifbarrera !